renderer/Material.cpp

```diff
@@ -1562,7 +1532,6 @@ void idMaterial::ParseStage( idLexer &src, const textureRepeat_t trpDefault ) {
 					newStage.vertexProgram = R_FindARBProgram( GL_VERTEX_PROGRAM_ARB, token.c_str() );
```

Comment:
I think the code for vertex program should be removed too.

Reply:
I just deleted that whole #ifdef block. Wouldn't have done much good anymore, anyway.